platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -1142,6 +1148,13 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 		spu_peripheral_config_secure(base_addresses[i], SPU_LOCK_CONF_LOCKED);
 	}
 
+	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
```

Comment:
```suggestion
	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed did. djdb ib s ib o ex is own[**_ djx I d in o b ex o s**B bdj z B B s. xis ib finder  no bz[**_ dubs in di w o de boo hare  boo in no ib w B isbiwbgjdobdjhzf s hi d hi s**_```suggestion
	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
```
_**_**]()**_**]()
```